cli/install/install.go

```diff
@@ -0,0 +1,631 @@
+			{"libunwind", "libunwind-devel"}, {"libicu", "libicu-devel"}}
+	} else if strings.Contains(osName, "Ubuntu") || strings.Contains(osName, "Debian") {
+		libraries = []Package{{"zlib1g-dev", "zlib1g-dev"}, {"libncurses5-dev", "libncurses5-dev"},
+			{"libreadline-dev", "libreadline-dev"}, {"libssl-dev", "libssl-dev"},
+			{"libunwind-dev", "libunwind-dev"}, {"libicu-dev", "libicu-dev"}}
+	} else {
+		answer, err := util.AskConfirm("Unknown OS, can't check if dependencies is installed.\n" +
+			"Procced, without checking?")
+		if !answer || err != nil {
+			return false
+		}
+		if answer {
+			return true
+		}
+	}
+	// Check that all dependencies are installed.
+	for i := 0; i < len(programs); i++ {
+		if !isProgrammInstalled(programs[i].sysName) {
+			miss_count++
+			missing_pack += " " + programs[i].installName
+		}
+	}
+	for i := 0; i < len(pyPackages); i++ {
+		if !isPythonPackageInstalled(pyPackages[i].sysName) {
+			miss_count++
+			missing_pack_python += " " + pyPackages[i].installName
+		}
+	}
+	for i := 0; i < len(libraries); i++ {
+		if !isLibraryInstalled(libraries[i].sysName) {
+			miss_count++
+			missing_pack += " " + libraries[i].installName
+		}
+	}
+	if miss_count != 0 {
+		log.Error("The operation requires some dependencies.")
+		log.Error("Missing packages: " + missing_pack + " " + missing_pack_python)
+		if osName == "darwin" || strings.Contains(osName, "freebsd") {
+			log.Error("You can install them by running commands:")
+			log.Error("brew install" + missing_pack)
+			log.Error("pip3 install" + missing_pack_python)
+		} else if strings.Contains(osName, "CentOs") {
+			log.Error("You can install them by running command:\n sudo yum install" + missing_pack)
+		} else if strings.Contains(osName, "Ubuntu") || strings.Contains(osName, "Debian") {
+			log.Error("You can install them by running command:\n sudo apt install" + missing_pack)
+		}
+		log.Error("Usage: tt install -f if You already have those packages installed")
+		return false
+	}
+	return true
+}
+
+// checkExisting checks if programm is already installed.
+func checkExisting(version string, dst string) bool {
+	if _, err := os.Stat(dst + "/" + version); os.IsNotExist(err) {
+		return false
+	} else {
+		return true
+	}
+}
+
+// executeCommand executes programm with given args in verbose or quiet mode.
+func executeCommand(programm string, isVerbose bool, logFile *os.File, path string,
+	args ...string) error {
+	cmd := exec.Command(programm, args...)
+	if isVerbose {
+		cmd.Stdout = os.Stdout
+		cmd.Stderr = os.Stderr
+	} else {
+		cmd.Stdout = logFile
+		cmd.Stderr = logFile
+	}
+	if path == "" {
+		path, _ = os.Getwd()
+	}
+	cmd.Dir = path
+	err := cmd.Start()
+	if err != nil {
+		return err
+	}
+	err = cmd.Wait()
+	return err
+}
+
+// changeSymlink changes symlink.
+func changeSymlink(version string, dst string, programm string) error {
+	os.Remove(dst + "/" + programm)
+	err := os.Symlink(version, dst+"/"+programm)
```

Comment:
Why not use `filepath.Join()`? (And you will avoid all missed whitespaces between operands and operators of strings concatenation)